server/plugin.go

```diff
@@ -757,3 +791,49 @@ func (p *Plugin) runMetricsUpdaterTask(store store.Store, updateMetricsTaskFrequ
 	metricsUpdater()
 	model.CreateRecurringTask("metricsUpdater", metricsUpdater, updateMetricsTaskFrequency)
 }
+
+func (p *Plugin) OnSharedChannelsPing(rc *model.RemoteCluster) bool {
+	p.API.LogDebug("Ping received", "remote", rc.DisplayName, "pluginID", pluginID)
+	return true
+}
+
+func (p *Plugin) OnSharedChannelsAttachmentSyncMsg(fi *model.FileInfo, post *model.Post, rc *model.RemoteCluster) error {
+	p.API.LogDebug("SyncMsg attachment", "fileInfo", fi, "post", post, "remote", rc.DisplayName, "pluginID", pluginID)
```

Comment:
Propose not logging this: will be too spammy and not useful.

Reply:
Done